static/admin/config.yml

```diff
@@ -258,6 +258,39 @@ collections:
                         },
                     ],
             }
+          - {
+                label: "Genomic Characterization",
+                name: "genomic_characterization",
+                widget: "object",
+                required: false,
```

Comment:
Our issue criteria specified that `collapsed: true` should be added, probably valid here as well?

Reply:
@rugeli can you add this change?